src/doc/rustc/src/platform-support/x86_64-asan-linux-gnu.md

```diff
@@ -0,0 +1,55 @@
+# `x86_64-asan-linux-gnu`
+
+**Tier: 3**
+
+Target mirroring `x86_64-unknown-linux-gnu` with AddressSanitizer enabled by
+default.
+The goal of this target is to allow shipping ASAN-instrumented standard
+libraries through rustup, enabling a fully instrumented binary without requiring
+nightly features (build-std).
```

Comment:
Just a note, to ship libstd this will have to become a tier 2 target (without host tools).